CHANGELOG.rst

```diff
@@ -1,6 +1,77 @@
 Changelog
 =========
 
+0.9.0 (2019-03-13)
+------------------
+
+Major features:
+
+- Support for running MLflow Projects in Docker containers. This allows users to include non-Python dependencies in their project environments and provides stronger isolation when running projects. See the `Projects documentation <https://mlflow.org/docs/latest/projects.html>`_ for more information. (#555, @marcusrehm; #819, @mparkhe; #970, @dbczumar)
+- SqlAlchemyStore: new backend store implementation using SQLAlchemy for MLflow Tracking Server. Support for a scalable and performant backend store was one of the top community requests. This community contribution feature enables users to connect to local or remote SQLAlchemy compatible databases (currently supported flavors include MySQL, PostgreSQL, SQLite, and MS SQL) and is compatible with file backed store. See the `Tracking Store documentation <https://mlflow.org/docs/latest/tracking.html#storage>`_ for more information. (#756, @AndersonReyes; #800, #844, #847, #848, #860, #868, #975, @mparkhe; #980, @dbczumar)
+- Simplified Python model customization. Custom preprocessing and postprocessing logic, as well as data dependencies, can be easily included in models with the python_function flavor using updated mlflow.pyfunc Python APIs. For more information, see the `Custom Python Models documentation <https://mlflow.org/docs/latest/models.html#custom-python-models>`_. (#791, #792, #793, #830, #910, @dbczumar)
+- Plugin systems allowing third party libraries to extend MLflow functionality. The `proposal document <https://gist.github.com/zblz/9e337a55a7ba73314890be68370fa69a>`_ gives the full detail of the three main changes: 
+
+  - Additional providers of tracking stores can be registered using the ``mlflow.tracking_store`` entrypoint. (#881, @zblz)
+  - Additional providers of artifact repository can be registered using the ``mlflow.artifact_repository`` entrypoint. (#882, @mociarain)
+  - The logic generating run metadata from the run context (e.g. ``source_name``, ``source_version``) has been refactored into an extendable system of run context providers. Plugins can register additional providers using the `mlflow.run_context_provider` entrypoint, which add to or overwrite tags set by the base library. (#913, #926, #930, #978, @acroz)
+
+- Support for authentication to the tracking server in the R client. Now R users can connect to secure tracking servers via credentials set in environment variables, or provide custom plugins for setting the credentials. As an example, in this release is the Databricks plugin which can detect existing Databricks credentials to allow users to connect to the Databricks tracking server. (#938, #959, #992, @tomasatdatabricks)
```

Comment:
tracking server -> Tracking Server

Now R users can connect to secure tracking servers via credentials -> 

Now you can connect to secure Tracking Servers using credentials

As an example, in this release is the Databricks plugin which can detect existing Databricks credentials to allow users to connect to the Databricks tracking server. ->

As an example, this release contains a Databricks plugin that can detect existing Databricks credentials to allow you to connect to the Databricks Tracking Server.

Reply:
This also supports generic HTTP auth, right? We should mention that too since many people just add standard HTTP auth in front of the tracking server.